### PR TITLE
Astro CLI is now on homebrew core

### PR DESCRIPTION
**Pull request by sunkickr**
all instances of `brew install astronomer/tap/astro` can now be changed to `brew install astro` because homebrew-core is now hooked up to our local tap

### CODE REVIEW

**Comment by sunkickr** [comment]
Do not merge this hasn't been tested yet for CI/CD

**Comment by sunkickr** [comment]
Do not merge this hasn't been tested yet for CI/CD

**Review by paolaperaza — APPROVED** [review]
LGTM! Thanks @sunkickr. Merge this whenever you're ready.

File: software_versioned_docs/version-0.27/cli-reference.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ The Astronomer CLI installation process requires [Docker](https://www.docker.com
 If you have Homebrew installed, run:
 
 ```sh
-brew install astronomer/tap/astro
+brew install astro@0.27.4
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.27/cli-reference.md:34]:
```suggestion
brew install astro@0.27
```

File: software_versioned_docs/version-0.27/cli-reference.md [hunk 2]
```diff
@@ -31,13 +31,13 @@ The Astronomer CLI installation process requires [Docker](https://www.docker.com
...
 ```
 
 To install a specific version of the Astro CLI, you'll have to specify `@major.minor.patch`. To install v0.16.1, for example, run:
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.27/cli-reference.md:37]:
```suggestion
This installs the latest patch version of Astro CLI v0.27. To install a specific version of the Astro CLI, specify a version in this format:
```

File: software_versioned_docs/version-0.23/cli-reference.md [hunk 3]
```diff
@@ -30,13 +30,13 @@ The Astronomer CLI installation process requires [Docker](https://www.docker.com
 If you have Homebrew installed, run:
 
 ```sh
-brew install astronomer/tap/astro
+brew install astro@0.23.4
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.23/cli-reference.md:33]:
```suggestion
brew install astro@0.23
```

File: software_versioned_docs/version-0.23/upgrade-to-0-23.md [hunk 4]
```diff
@@ -123,7 +123,7 @@ curl -sSL https://install.astronomer.io | sudo bash -s -- v0.23.0
 To do so via Homebrew, run:
 
 ```sh
-brew install astronomer/tap/astro@0.23
+brew install astro@0.23.4
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.23/upgrade-to-0-23.md:126]:
```suggestion
brew install astro@0.23
```

File: software_versioned_docs/version-0.25/cli-quickstart.md [hunk 5]
```diff
@@ -89,7 +89,7 @@ To install the Astronomer CLI on a machine with an [Apple M1 chip](https://www.a
 3. Run the following command to install the Astronomer CLI:
 
     ```sh
-    arch -x86_64 /usr/local/Homebrew/bin/brew install astronomer/tap/astro
+    arch -x86_64 /usr/local/Homebrew/bin/brew install astronomer/tap/astro@0.25.5
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.25/cli-quickstart.md:92]:
```suggestion
    arch -x86_64 /usr/local/Homebrew/bin/brew install astro
```

**Review by paolaperaza — APPROVED** [review]

File: software_versioned_docs/version-0.16/quickstart.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ curl -ssl https://install.astronomer.io | sudo bash
 To install the latest version of the Astronomer CLI via [Homebrew](https://brew.sh/), run:
 
 ```bash
-brew install astronomer/tap/astro
+brew install astro@0.16.1
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.16/quickstart.md:42]:
```suggestion
brew install astro
```

File: software_versioned_docs/version-0.16/upgrade-to-0-23.md [hunk 7]
```diff
@@ -123,7 +123,7 @@ curl -sSL https://install.astronomer.io | sudo bash -s -- v0.23.0
 To do so via Homebrew, run:
 
 ```sh
-brew install astronomer/tap/astro@0.23
+brew install astro@0.23.4
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.16/upgrade-to-0-23.md:126]:
```suggestion
brew install astro@0.23
```

**Pull request merged**